### ISSUE TITLE
Rack becomes "unreachable" in Console briefly after app deployment

### ISSUE THREAD

**Issue by frankleng** [convox/rack#1238]
Wonder if this has to do with autoscaling. Convox currently scales up to 7, and then back down to 4 during app deployment. will test once #1230 is released and report back.

```
client: 20160911232453
server: 20160910125708 (console.convox.com)

rack params
NAME                     VALUE
Ami                      ****
ApiCpu                   128
ApiMemory                128
Autoscale                Yes
ClientId                 ***
ContainerDisk            10
Development              No
Encryption               Yes
ExistingVpc
InstanceBootCommand
InstanceCount            4
InstanceRunCommand
InstanceType             m4.large
InstanceUpdateBatchSize  1
Internal                 No
Key                      ****
Password                 ****
Private                  Yes
PrivateApi               No
Subnet0CIDR              10.0.1.0/24
Subnet1CIDR              10.0.2.0/24
Subnet2CIDR              10.0.3.0/24
SubnetPrivate0CIDR       10.0.4.0/24
SubnetPrivate1CIDR       10.0.5.0/24
SubnetPrivate2CIDR       10.0.6.0/24
SwapSize                 5
Tenancy                  default
VPCCIDR                  10.0.0.0/16
Version                  20160910125708
VolumeSize               50
```

**Comment by andykent** [comment]
We also saw exactly this behaviour after turning on Autoscaling earlier today.

**Comment by ddollar** [comment]
This should be fixed by #1230 and #1239
